docs/source/ecf.rst

```diff
@@ -1206,8 +1206,8 @@ timescales. It is also possible to plot
 'fn' (the nightside flux from a sinusoidal phase curve),
 'pc_offset' (the sinusoidal offset of the phase curve),
 'pc_amp' (the sinusoidal amplitude of the phase curve),
-'pc_offset2' (the second order sinusoidal offset of the phase curve), and
-'pc_amp2' (the second order sinusoidal amplitude of the phase curve).
+'offset_order2' (the second order sinusoidal offset of the phase curve), and
+'amp_order2' (the second order sinusoidal amplitude of the phase curve).
```

Comment:
Here (and perhaps also in the S6 template), we should make it clear that amp_order1 and offset_order1 are also supported.